CareKit/Connect/OCKConnectDetailViewController.m

```diff
@@ -181,11 +182,11 @@ - (void)contactInfoTableViewCellDidSelectConnection:(OCKContactInfoTableViewCell
             break;
             
         case OCKConnectTypeMessage:
-            [self sendMessageToNumber:cell.contact.messageNumber.stringValue];
+            [self sendMessageToNumber:cell.contact.messageNumber.stringValue withDelegate:self usingPresentingViewController:self];
             break;
             
         case OCKConnectTypeEmail:
-            [self sendEmailToAddress:cell.contact.emailAddress];
+            [self sendEmailToAddress:cell.contact.emailAddress withDelegate:self usingPresentingViewController:self];
```

Comment:
Is there a reason for making the presenting view controller a parameter?
Could you instead just use self inside the method?